dvc/repo/tree.py

```diff
@@ -142,7 +142,7 @@ def _walk(self, root, trie, topdown=True):
         files = []
 
         root_len = len(root.parts)
-        for key, out in trie.iteritems(prefix=root.parts):
+        for key, out in trie.iteritems(prefix=root.parts):  # noqa: B301
```

Comment:
False positive from `flake8-bugbear`. If it sees `iteritems`, it assumes it's a dict and is incompatible with Python3.